sdk/ml/azure-ai-ml/azure/ai/ml/entities/_registry/registry.py

```diff
@@ -8,26 +8,34 @@
 from pathlib import Path
 from typing import IO, AnyStr, Dict, List, Union
 
+from azure.ai.ml._restclient.v2022_10_01_preview.models import ManagedServiceIdentity as RestManagedServiceIdentity
+from azure.ai.ml._restclient.v2022_10_01_preview.models import (
+    ManagedServiceIdentityType as RestManagedServiceIdentityType,
+)
 from azure.ai.ml._restclient.v2022_10_01_preview.models import Registry as RestRegistry
+from azure.ai.ml._restclient.v2022_10_01_preview.models import RegistryProperties
 from azure.ai.ml._schema.registry.registry import RegistrySchema
 from azure.ai.ml._utils.utils import dump_yaml_to_file
 from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY, PARAMS_OVERRIDE_KEY
+from azure.ai.ml.entities._registry.identity import ManagedServiceIdentity
 from azure.ai.ml.entities._resource import Resource
 from azure.ai.ml.entities._util import load_from_dict
+from azure.ai.ml._utils._experimental import experimental
 
-from .registry_support_classes import RegistryRegionArmDetails
+from .registry_support_classes import RegistryRegionArmDetails, SystemCreatedStorageAccount
 
 YAML_REGION_DETAILS = "replication_locations"
```

Comment:
Let's decide on one parameter name for region details and unify between CLI schema and SDK class.

Reply:
Ok, we'll stick with replication_locations, the schema name. This was the name that was explicitly asked for by PM's, despite the different name used by swagger objects and currently the SDK.